Documentation/k8s-object.md

```diff
@@ -80,53 +80,63 @@ See the [Object Storage](client.md#object-storage) documentation for more steps
 ## Access External to the Cluster
 
 Rook sets up the object storage so pods will have access internal to the cluster. If your applications are running outside the cluster,
-you will need to setup an external service through a `NodePort`.
+you will need another level of setup. In this example we will create an ingress controller. You could also create a service based on a `NodePort`.
```

Comment:
can you also add `LoadBalancer`?. So "you will need another level of setup. In this example we will create an ingress controller. You could also create a service based on a `NodePort` or `LoadBalancer`."

Reply:
ok sounds good